ads/insticator.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function insticator(global, data) {
+  // validate passed data attributes
+  validateData(data, ['siteId', 'embedId']);
+
+  // create insticator markup
+  global.document.getElementById('c').appendChild(createTemplate(data.embedId));
+
+  // create ads and embed
+  createAdsAndEmbed(data.siteId, data.embedId);
+}
+
+// ------- HELPER VARIABLES ------- //
+// reusable URL references to ads, embed and the library
+// Don't forget to preconnect and prefetch as it's described in the docs
+// https://github.com/ampproject/amphtml/blob/master/ads/README.md
+const url = {content: '//d3lcz8vpax4lo2.cloudfront.net'};
+
+/**
+ * Create HTML template to be populated later
+ * @param {string} embedId The Unique Identifier of this particular Embed
+ * @return {element} HTML template
+ */
+function createTemplate(embedId) {
+  const template = document.createElement('template');
+  template./*OK*/innerHTML = `
```

Comment:
ugh looks like you need a space here for linter `template./*OK/ innerHTML`

Reply:
Adding a space breaks the code and throws a console error

Reply:
@calebcordry Can I maybe just remove the /*OK*/ stuff? I'm unfamiliar with that. What does it do?

Reply:
We forbid the use of `innerHTML` in our code base unless reviewed. The `OK` lets us know someone has approved it. I just cloned your PR and made the change, and it seems to be fine? 
![image](https://user-images.githubusercontent.com/16087874/66087577-d5cfd500-e52c-11e9-8f99-6370bcf2d461.png)

Reply:
Seems to be working for me now as well. Strange. 

This has now been updated.